index.js

```diff
@@ -9,16 +9,14 @@ const [DUMP_JSON_DIR, DUMP_RAW_DIR] = [DL_DIR + '/json', DL_DIR + '/raw'];
 const {OCTANT, MAX_LEVEL, DUMP_JSON, DUMP_RAW} = require('./lib/parse-command-line')(process.argv.slice(2));
 /****************************************************************/
 
-let execSync = require('child_process').execSync;
-const fs = require('fs');
+const fs = require('fs-extra');
 const decodeResource = require('./lib/decode-resource');
 const decodeTexture = require('./lib/decode-texture');
 const getUrl = require('./lib/get-url');
 
 if (DUMP_JSON || DUMP_RAW) {
-	DUMP_JSON && execSync(`mkdir -p ${DUMP_JSON_DIR}`);
-	DUMP_RAW && execSync(`mkdir -p ${DUMP_RAW_DIR}`);
-	execSync = () => {};
```

Comment:
needs replacement for `execSync = () => {}`

Reply:
mkdirSync also needs to be recursive i think

Reply:
I am not sure if I understand the purpose of execSync if we perform all operations via fs-extra. What do you need a replacement for?

I have changed both mkdirSync and mkdirpSync to ensureDirSync which actually has its own [documentation](https://github.com/jprichardson/node-fs-extra/blob/master/docs/ensureDir-sync.md).